template.tpl

```diff
@@ -587,6 +587,7 @@ ___TEMPLATE_PARAMETERS___
             "type": "TEXT"
           },
           {
+            "help": "Please note that a hardcoded value in the tag will always convert an input into a string. If you want to use other types, please create a GTM variable, specifically a Data Layer Variable. This will accurately capture the types you've specified.",
```

Comment:
Does this work? @sahava 
Since the note is only specific to the value filed. I added a help note here.

Reply:
@yuhao900914 No, that won't work, individual fields in the table don't have the option of including Help text. That's why I suggested it should be added to the `identifyLabel` text.

Best way to test it is to create a new template in a Google Tag Manager container (Templates -> Tags -> New) and then import the .tpl file using the template editor settings menu option. That way you can preview what the template looks like and test it too.

Reply:
Added the note into `identifyLabel`
<img width="826" alt="Screen Shot 2023-06-14 at 10 41 09 AM" src="https://github.com/amplitude/amplitude-browser-sdk-gtm-template/assets/13028329/0f25524f-f063-4761-aaf9-73b894935b6f">